src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/TestCustomResourceLoader.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Resources;
+
+namespace Uno.UI.Tests.Windows_UI_Xaml_Markup.XamlReaderTests
+{
+	internal class TestCustomXamlResourceLoader : CustomXamlResourceLoader
+	{
+		internal Dictionary<string, object> TestCustomResources { get; set; } = new Dictionary<string, object>();
+
+		protected override object GetResource(string resourceId, string objectType, string propertyName, string propertyType)
+		{
```

Comment:
Let's assert on these values in the test, to make sure we're not regressing. For the `propertyType` value, since the types exposed are the ones from WinRT and we will not be replicating this at this time, let's use the .NET types.

```suggestion
        public string LastResourceId { get; set; }
        public string LastObjectType { get; set; }
        public string LastPropertyName { get; set; }
        public string LastPropertyType { get; set; }
        
		protected override object GetResource(string resourceId, string objectType, string propertyName, string propertyType)
		{
		    LastResourceId = resourceId;
		    LastObjectType = objectType;
		    LastPropertyName = propertyName;
		    LastPropertyType = propertyType;
```